README.md

```diff
@@ -130,3 +130,18 @@ Overly complex structures and unnecessary code can be avoided by effective refac
 * Overly-nested and long functions
 * Long parameter lists
 
+
+## Code Reviews
+### [Evaluation Feedback process](https://smartbear.com/learn/code-review/guide-to-code-review-process/)
+* #### Use checklists
+Checklists are the most effective way to eliminate frequently made errors. Code review checklists also provide team members with clear expectations for each type of review and can be helpful to track for reporting and process improvement purposes.
+
+* #### Establish a process for fixing defects found
+The best way to ensure that defects are fixed is to use a collaborative code review tool that allows reviewers to log bugs, discuss them with the author, and approve changes in the code.
```

Comment:
Is there a code review tool you have in mind? if so mention it, and if not you should probably research one to give people advice so the whole company use the same code review tool

Reply:
My topic was specifically code review evaluation or steps after the actual code review itself so I couldn't really find any tools or practices anywhere online. Should I change my heading to include review tools aswell or just leave it as is?